test: add coverage

GuillaumeCisco opened the pull request:
## Pull Request checklist:

- [x] Write an explicit title for the Pull Request, following [Conventional Commits specification](https://www.conventionalcommits.org)
- [ ] Create automatic tests
- [x] No automatic tests failures
- [x] Test manually the implemented changes
- [x] Review my own code (indentation, syntax, style, simplicity, readability)
- [x] Wonder if you can improve the existing code

arnaudbesnier commented on Gemfile:
```diff
@@ -33,3 +33,4 @@ gem 'base32', '0.3.2'
 gem 'rotp', '3.1'
 gem 'httparty', '0.13.7'
 gem 'ipaddress', '0.8.3'
+gem 'simplecov', require: false, group: :test
```
You should put this gem definition in "test" group at line 19, to ease the readability.

arnaudbesnier commented on Gemfile.lock:
```diff
@@ -135,20 +137,25 @@ GEM
+    tzinfo (1.2.8)
       thread_safe (~> 0.1)
-    useragent (0.16.5)
+    useragent (0.16.10)
```
Why all those updates?
This is a quite hazardous operation (especially when you upgrade to new major versions)
And I don't see the relation with the coverage impllementation.

GuillaumeCisco replied:
Looks like `bundle install` auto update these dependencies

arnaudbesnier commented:
Is this coverage run by the CI?

GuillaumeCisco commented:
> Is this coverage run by the CI?

no, it is still a work in progress to show what could be done

GuillaumeCisco commented:
It should be ok now.
I cleaned and added [codecov](https://app.codecov.io/gh/ForestAdmin/forest-rails/branch/chore-coverage) for travis.
Maybe we also need to bind code climate to this repository :)

arnaudbesnier commented on README.md:
```diff
@@ -1,6 +1,8 @@
 [![Gem](https://badge.fury.io/rb/forest_liana.svg)](https://badge.fury.io/rb/forest_liana)
-[![CI status](https://travis-ci.org/ForestAdmin/forest-rails.svg?branch=devel)](https://travis-ci.org/ForestAdmin/forest-rails)
+[![CI status](https://travis-ci.org/ForestAdmin/forest-rails.svg?branch=master)](https://travis-ci.org/ForestAdmin/forest-rails)
+[![codecov](https://codecov.io/gh/ForestAdmin/forest-rails/branch/master/graph/badge.svg)](https://codecov.io/gh/ForestAdmin/forest-rails)
```
Why do we use both Codecov and Code Climate? We should clean the Codecov badge.

GuillaumeCisco replied:
We should remove it :)

arnaudbesnier reviewed (approved):
Looks great! 🎖️ 
Let's wait the last commit to be CI-ed to see if the coverage is still reported.
